squirrel_safety/launch/squirrel_safety.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <node name="squirrel_safety_node" pkg="squirrel_safety" type="squirrel_safety_node.py" output="screen" respawn="true">
+    <param name="airskin_topic" value="/arm_bumper" type="string"/>
+    <param name="bumper_topic" value="/bumper" type="string"/>
+    <param name="wrist_topic" value="/wrist/wrist_safety" type="string"/>
```

Comment:
I think the topic name is `/wrist/wrist_bumper` https://github.com/squirrel-project/squirrel_driver/blob/0d963d9109add42329ad423ae0a857bc2176d100/wrist_safety/src/wrist_safety.cpp#L34